doc news: add 15.1.1 entry (#2346)
groonga/groonga: doc/source/news/15.md

# News - 15 series

(release-15-1-1)=

## Release 15.1.1 - 2025-05-29

This release updates TokenMecab to preserve user-defined entries with spaces as
single tokens and adds an Arch Linux package.

### Improvements

#### {doc}`/reference/tokenizers/token_mecab`: Fix unintended splitting of user-defined words with spaces

> Previously, TokenMecab internally treated MeCab’s `-Owakati` mode output as

Author: fix: [9041796](https://github.com/groonga/groonga/pull/2346/commits/9041796279f117a2fd7679d642313cbc040f72b7) アンカーをおいてリンクにしました。

space-delimited tokens, causing user-defined dictionary entries containing
spaces (e.g., "search engine") to be split into separate tokens like "search"

> and "engine". This release updates TokenMecab to use MeCab’s `-F%m\n`

Reviewer: ```suggestion
and "engine". This release updates TokenMecab to use MeCab's `-F%m\n`
```

（実装の詳細はここでは説明しなくてもいい気はするけど。。。）

newline-delimited output, ensuring that entries with embedded spaces remain as
single tokens.

#### Added support for Arch Linux package

We provided the package of Arch Linux.

> This is provided via the Arch User Repository and delivers the latest Groonga

Reviewer: すでにAURにはGroongaのパッケージがあったので、うちらが始めましたみたいな言い方はよくないっす。

書くならArch Linuxユーザー向けのインストールドキュメントを用意して、インストール方法をまとめました、くらいじゃない？

# News - 15 series

(release-15-1-1)=

## Release 15.1.1 - 2025-06-02

This release updates TokenMecab to preserve user-defined entries with spaces as
single tokens.

### Improvements

#### {doc}`/reference/tokenizers/token_mecab`: Fix unintended splitting of user-defined entries with spaces

Previously, {ref}`token-mecab` split user-defined entries containing spaces
(e.g., "search engine") into separate tokens ("search" and "engine"). This
release fixes this issue, so entries with embedded spaces are now preserved and
handled as single tokens like "search engine" as follows.

```
tokenize TokenMecab "search engine" --output_pretty yes
[
  [
    0,
    1748413131.972704,
    0.0003032684326171875
  ],
  [
    {
      "value": "search engine",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
```

### Fixes

> #### Fixed wide ranging typos across documentation

Reviewer 2: ```suggestion
#### Fixed many typos in documentation
```

> Corrected typos through a series of PRs, each addressing words beginning with

Reviewer 2: ```suggestion
GH-2333, GH-2334, GH-2335, GH-2336, GH-2337, GH-2338

Corrected typos through a series of PRs, each addressing words beginning with
```
